packages/webapp/src/components/LeftNavBar.tsx

```diff
@@ -220,7 +220,7 @@ export default function LeftNavBar(props: LeftNavBarProps) {
                                 <span className="items-center w-32 text-center text-gray-400 justify-center text-left text-sm truncate">{email}</span>
                             </div>
                             <EllipsisHorizontalIcon className="flex h-5 w-5 ml-3 text-gray-400 cursor-pointer" />
-                            {isCloud() && showUserSettings && (
+                            {(isCloud() || isEnterprise()) && showUserSettings && (
```

Comment:
can isCloud and isEnterprise be included in the `showUserSettings` logic?

Reply:
`showUserSettings` is just a useState prop so there is actual no logic there just a boolean. The entire logic piece could be abstracted out potentially but I don't see much of an advantage there.

Reply:
showUserSettings alone is then not enough to determine if user settings should or should not be shown, which seems weird to me because that the main purpose of this boolean state. You are the react expert though so up to you